src/python_testing/TC_EPM_2_1.py

```diff
@@ -0,0 +1,98 @@
+            TestStep("9", "TH reads ApparentCurrent attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of 0 to 2^62."),
+            TestStep("10", "TH reads ActivePower attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("11", "TH reads ReactivePower attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("12", "TH reads ApparentPower attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("13", "TH reads RMSVoltage attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("14", "TH reads RMSCurrent attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("15", "TH reads RMSPower attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+            TestStep("16", "TH reads Frequency attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of 0 to 1000000."),
+            TestStep("17", "TH reads HarmonicCurrents attribute. Verify that the DUT response contains a list of HarmonicMeasurementStruct entries."),
+            TestStep("18", "TH reads HarmonicPhases attribute. Verify that the DUT response contains a list of HarmonicMeasurementStruct entries."),
+            TestStep("19", "TH reads PowerFactor attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -10000 to 10000."),
+            TestStep("20", "TH reads NeutralCurrent attribute. Verify that the DUT response contains either null or an int64 value. Value has to be between a range of -2^62 to 2^62."),
+        ]
+
+        return steps
+
+    @async_test_body
+    async def test_TC_EPM_2_1(self):
+
+        self.step("1")
+        # Commission DUT - already done
+
+        self.step("2")
+        power_mode = await self.read_epm_attribute_expect_success("PowerMode")
+        logger.info(f"Rx'd PowerMode: {power_mode}")
+        asserts.assert_not_equal(power_mode, Clusters.ElectricalPowerMeasurement.Enums.PowerModeEnum.kUnknown,
+                                 "PowerMode must not be Unknown")
+
+        self.step("3")
+        number_of_measurements = await self.read_epm_attribute_expect_success("NumberOfMeasurementTypes")
+        logger.info(f"Rx'd NumberOfMeasurementTypes: {number_of_measurements}")
+        asserts.assert_greater(number_of_measurements, 1,
+                               "NumberOfMeasurementTypes must be > 1")
```

Comment:
it in "min 1", so >0
```suggestion
        asserts.assert_greater(number_of_measurements, 0,
                               "NumberOfMeasurementTypes must be > 0")
```

Reply:
changed to >=